etc/helm/pachyderm/values.yaml

```diff
@@ -219,7 +219,7 @@ determined:
   # Distributed tasks with sizes that are not divisible by `maxSlotsPerPod` are never scheduled. If
   # you have a cluster of different size nodes (e.g., 4 and 8 GPUs per node), set `maxSlotsPerPod` to
   # the greatest common divisor of all the sizes (4, in that case).
-  maxSlotsPerPod:
+  maxSlotsPerPod: 0
```

Comment:
will this default still allow for a helm failure to occur. from determined this is required i think and 0 should not be a default

Reply:
I commented this out instead of adding a default. That way the user hit's determined's error message directing them to manually set this value.